android/src/main/java/org/reactnative/maskedview/RNCMaskedView.java

```diff
@@ -42,7 +42,10 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
     super.onLayout(changed, l, t, r, b);
 
     if (changed) {
-      this.mBitmapMask = getBitmapFromView(getChildAt(0));
+      View maskView = getChildAt(0);
+      maskView.setAlpha(1);
```

Comment:
You can also setVisibility to View.INVISIBLE here and resign from manipulating the alpha.
```suggestion
      maskView.setVisibility(View.INVISIBLE);
```

Reply:
Good idea. I've changed that now 👍

Reply:
You can also set View.INVISIBLE permanently on the mask. There is no need to set it back to View.VISIBLE as getBitmapFromView should work with View.INVISIBLE from what I have tested.

Anyway, it is fine this way as well of course.